### DISCUSSION TITLE
Inappropriate action by "users"

### DISCUSSION THREAD

**Discussion by MitkoVtori** [community/community#42832] [Discussions] [answered]
### Select Topic Area

General

### I don't know If someone notice, but on my email I recieve messages from my discussions, and today I recieved three similiar messages by three different users on three different discussions in three different repositories(including community/community).

**The messages are Inappropriate, after that there's always the same link next to it(probably some sort of scam link) and then a lot of random users tagged.**

#### But the main problem is, even though they are delated, I still see them on my email.

I don't know if this is a bunch of random users, or a script, or if you click on the link does it hacks you so they used hacked profiles, But I would like to not recieve such a notifications.

Thank you for your time! 💖

@github-staff I would be thankful If you can help somehow.

**Comment by Akash1134** [comment, marked as answer]
Hi @MitkoVtori, GitHub is responding to concerns expressed by community members regarding recent spam posts & comments. Thanks for sharing and we appreciate your patience.
